crypto/types/src/types.rs

```diff
@@ -643,61 +666,55 @@ pub type PacketTag = [u8; PACKET_TAG_LENGTH];
 
 /// Represents a secp256k1 public key.
 /// This is a "Schrödinger public key", both compressed and uncompressed to save some cycles.
+///
+/// ```rust
+/// # use hex_literal::hex;
+/// # use hopr_crypto_types::prelude::*;
+/// # use k256::ecdsa::VerifyingKey;
```

Comment:
```suggestion
/// use hex_literal::hex;
/// use hopr_crypto_types::prelude::*;
/// use k256::ecdsa::VerifyingKey;
```

Reply:
In docstrings, it is necessary to make the code compile but these lines are excluded from rendering for better observability